h2o-core/src/main/java/water/fvec/Frame.java

```diff
@@ -1787,7 +1787,7 @@ public int available() {
       Chunk anyChunk = _curChks[0];
 
       // Case 3:  Out of data.
-      if (anyChunk._start + _chkRow == anyChunk._vec.length()) {
+      if (anyChunk._start + _chkRow >= anyChunk._vec.length()) {
```

Comment:
A bug found. The reader enclosing InputStream, calling the `available` method might actually call the `available` method multiple times. This is a completely valid scenario. For the first time it is called, this stream returns `-1` correctly. Yet the chunk row index increments as well with the call and the state of the stream did not pass the `==` check. Therefore changed to `>=`. 

Naturally tested by the newly introduced test by Apache Common's IOUtils. Taking this change away will crash the test :wink: